ModelCatalogueCorePlugin/grails-app/assets/javascripts/modelcatalogue/core/ui/catalogueElementTreeviewItem.coffee

```diff
@@ -180,6 +188,20 @@ angular.module('mc.core.ui.catalogueElementTreeviewItem', ['mc.util.names', 'mc.
         $scope.collapseOrExpand()
         $scope.treeview.select(element)
 
+      $scope.metadataOccurrencesToAsterisk = (element) ->
+        # only Data Element with some metadata and defined relationship
+        if element.$$metadata?.values? && element.getElementTypeName() == 'Data Element' && element.$$relationship
```

Comment:
yeah, that sound reasonable, just instead of `element.getElementTypeName() == 'Data Element'` there should be `element.isInstanceOf('dataElement')`. what about child data classes? should they show occurrence as well?